infra/Pos/Communication/Relay/Types.hs

```diff
@@ -51,3 +56,17 @@ data RelayContext = RelayContext
     { _rlyIsPropagation    :: !Bool
     , _rlyPropagationQueue :: !RelayPropagationQueue
     }
+
+makeLenses ''RelayContext
```

Comment:
I went for whichever notation worked best with existing field names. Doesn't really matter because those lenses are not exported (and not meant to be exported).